next_paint_hack.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
+
+var global = this;
```

Comment:
Won't this just be window?

Use const

Reply:
If we just use window as the global variable container, will it contaminate the window's namespace?